atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/ConnectionConfig.java

```diff
@@ -167,7 +167,7 @@ public HikariConfig getHikariConfig() {
 
         config.setPoolName(getConnectionPoolName());
         config.setRegisterMbeans(true);
-        config.setMetricRegistry(SharedMetricRegistries.getOrCreate("com.palantir.metrics"));
+        config.setMetricRegistry(SharedMetricRegistries.tryGetDefault());
```

Comment:
If no default metric registry has been set, this will return null, and Hikari just won't produce metrics. This should be fine since AFAICT nothing is consuming metrics the `com.palantir.metrics` metric registry.

Reply:
I don't think you can do this, this will instantly break metrics for lots of internal projects. Internal projects currently rely on the fact that they configure this named shared registry for metrics.

Reply:
For the record as well, we have lots of internal dashboards and alert definitions that consume these metrics.